linkerd/docs/config.md

```diff
@@ -159,16 +167,27 @@ Each router must be configured as an object with the following params:
   * *trees* -- Optional.  Size of the tree cache.  (default: 100)
   * *bounds* -- Optional.  Size of the bound cache.  (default: 100)
   * *clients* -- Optional.  Size of the client cache.  (default: 10)
-<a name="response_classifier"></a>
 * *responseClassifier* -- Optional. A
-  (sometimes protocol-specific) [response classifier](response_classifier.md)
+  (sometimes protocol-specific) [response classifier](#http-response-classifiers)
   that determines which responses should be considered failures and, of those,
-  which should be considered [retryable](retries.md).
+  which should be considered [retryable](#retries).
   (default: _io.l5d.nonRetryable5XX_)
 
 <a name="basic-server-params"></a>
 ### Basic server parameters
 
+```yaml
+servers:
+- port: 8080
+  ip: 0.0.0.0
+  tls:
+    certPath: /foo/cert.pem
+    keyPath: /foo/key.pem
+  maxConcurrentRequests: 1000
+  announce:
+    - /#/io.l5d.fs/web
```

Comment:
Nothing wrong with this but we don't currently have an announcer that can announce `/#/io.l5d.fs` names.  So far we just have one for `/#/io.l5d.serversets`.